### ISSUE TITLE
Mutagen status command can't find the session

### ISSUE THREAD

**Issue by deviantintegral** [drud/ddev#3129]
**Describe the bug**

After starting or restarting with mutagen enabled, mutagen status isn't available. However, mutagen _is_ working, and properly syncing files in both directions.

```
$ ddev restart
Restarting project marketing-library...
Stopping ddev-marketing-library-web ...
Stopping ddev-marketing-library-dba ...
Stopping ddev-marketing-library-db  ...
Stopping ddev-marketing-library-web ... done
Stopping ddev-marketing-library-dba ... done
Stopping ddev-marketing-library-db  ... done
Removing container: ddev-marketing-library-web
Removing container: ddev-marketing-library-dba
Removing container: ddev-marketing-library-db
Pushed mkcert rootca certs to ddev-global-cache/mkcert
Building db
Building web
Creating ddev-marketing-library-db ...
Creating ddev-marketing-library-db ... done
Creating ddev-marketing-library-dba ...
Creating ddev-marketing-library-web ...
Creating ddev-marketing-library-dba ... done
Creating ddev-marketing-library-web ... done
Starting mutagen sync process...
Using mutagen config file /Users/andrew/workspace/client/marketing-library/.ddev/mutagen.yml
Mutagen sync completed in 4.2s.
For details on sync status 'ddev mutagen status marketing-library --verbose'
Creating ddev-router ...
Creating ddev-router ... done
Ensuring write permissions for marketing-library
Existing settings.php file includes settings.ddev.php
Ensuring write permissions for marketing-library
Restarted marketing-library
Your project can be reached at https://marketing-library.ddev.site https://127.0.0.1:49710

$ ddev mutagen status marketing-library --verbose
Mutagen not working:
Error: unable to locate requested sessions: specification "marketing-library" did not match any sessions
```

**To Reproduce**

- Run `ddev config --mutagen-enabled` in a Drupal 9 project.
- Run `ddev start`
- Try to run `ddev mutagen status marketing-library --verbose`

**Expected behavior**

- The status command should work!

**Version and configuration information (please complete the following information):**

- macOS 11.5.1, Docker for Mac 3.5.2.
```
$ ddev version
DDEV-Local version	v1.18.0-alpha1
architecture      	amd64
db                	drud/ddev-dbserver-mariadb-10.3:20210709_mariadb_versions
dba               	phpmyadmin:5
ddev-ssh-agent    	drud/ddev-ssh-agent:v1.17.0
docker            	20.10.7
docker-compose    	1.29.2
mutagen           	0.12.0-beta4
os                	darwin
router            	drud/ddev-router:v1.17.6
web               	drud/ddev-webserver:20210714_bullseye_web_image
```

```
name: marketing-library
type: drupal9
docroot: web
php_version: "7.4"
webserver_type: nginx-fpm
router_http_port: "80"
router_https_port: "443"
xdebug_enabled: false
additional_hostnames: []
additional_fqdns: []
mariadb_version: "10.3"
mysql_version: ""
nfs_mount_enabled: false
mutagen_enabled: true
use_dns_when_possible: true
composer_version: ""
web_environment: []


# This config.yaml was created with ddev version v1.18.0-alpha1
# webimage: drud/ddev-webserver:20210714_bullseye_web_image
# dbimage: drud/ddev-dbserver-mariadb-10.3:20210709_mariadb_versions
# dbaimage: phpmyadmin:5
# However we do not recommend explicitly wiring these images into the
# config.yaml as they may break future versions of ddev.
# You can update this config.yaml using 'ddev config'.

# Key features of ddev's config.yaml:

# name: <projectname> # Name of the project, automatically provides
#   http://projectname.ddev.site and https://projectname.ddev.site

# type: <projecttype>  # drupal6/7/8, backdrop, typo3, wordpress, php

# docroot: <relative_path> # Relative path to the directory containing index.php.

# php_version: "7.4"  # PHP version to use, "5.6", "7.0", "7.1", "7.2", "7.3", "7.4" "8.0"

# You can explicitly specify the webimage, dbimage, dbaimage lines but this
# is not recommended, as the images are often closely tied to ddev's' behavior,
# so this can break upgrades.

# webimage: <docker_image>  # nginx/php docker image.
# dbimage: <docker_image>  # mariadb docker image.
# dbaimage: <docker_image>

# mariadb_version and mysql_version
# ddev can use many versions of mariadb and mysql
# However these directives are mutually exclusive
# mariadb_version: 10.2
# mysql_version: 8.0

# router_http_port: <port>  # Port to be used for http (defaults to port 80)
# router_https_port: <port> # Port for https (defaults to 443)

# xdebug_enabled: false  # Set to true to enable xdebug and "ddev start" or "ddev restart"
# Note that for most people the commands
# "ddev xdebug" to enable xdebug and "ddev xdebug off" to disable it work better,
# as leaving xdebug enabled all the time is a big performance hit.

# xhprof_enabled: false  # Set to true to enable xhprof and "ddev start" or "ddev restart"
# Note that for most people the commands
# "ddev xhprof" to enable xhprof and "ddev xhprof off" to disable it work better,
# as leaving xhprof enabled all the time is a big performance hit.

# webserver_type: nginx-fpm  # or apache-fpm

# timezone: Europe/Berlin
# This is the timezone used in the containers and by PHP;
# it can be set to any valid timezone,
# see https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
# For example Europe/Dublin or MST7MDT

# composer_version: ""
# if composer_version:"" it will use the current ddev default composer release.
# It can also be set to "1", to get most recent composer v1
# or "2" for most recent composer v2.
# It can be set to any existing specific composer version.
# After first project 'ddev start' this will not be updated until it changes

# additional_hostnames:
#  - somename
#  - someothername
# would provide http and https URLs for "somename.ddev.site"
# and "someothername.ddev.site".

# additional_fqdns:
#  - example.com
#  - sub1.example.com
# would provide http and https URLs for "example.com" and "sub1.example.com"
# Please take care with this because it can cause great confusion.

# upload_dir: custom/upload/dir
# would set the destination path for ddev import-files to custom/upload/dir.

# working_dir:
#   web: /var/www/html
#   db: /home
# would set the default working directory for the web and db services.
# These values specify the destination directory for ddev ssh and the
# directory in which commands passed into ddev exec are run.

# omit_containers: [db, dba, ddev-ssh-agent]
# Currently only these containers are supported. Some containers can also be
# omitted globally in the ~/.ddev/global_config.yaml. Note that if you omit
# the "db" container, several standard features of ddev that access the
# database container will be unusable.

# nfs_mount_enabled: false
# Great performance improvement but requires host configuration first.
# See https://ddev.readthedocs.io/en/stable/users/performance/#using-nfs-to-mount-the-project-into-the-container

# mutagen_enabled: false
# Experimental performance improvement using mutagen asynchronous updates.
# See https://ddev.readthedocs.io/en/latest/users/performance/#using-mutagen

# fail_on_hook_fail: False
# Decide whether 'ddev start' should be interrupted by a failing hook

# host_https_port: "59002"
# The host port binding for https can be explicitly specified. It is
# dynamic unless otherwise specified.
# This is not used by most people, most people use the *router* instead
# of the localhost port.

# host_webserver_port: "59001"
# The host port binding for the ddev-webserver can be explicitly specified. It is
# dynamic unless otherwise specified.
# This is not used by most people, most people use the *router* instead
# of the localhost port.

# host_db_port: "59002"
# The host port binding for the ddev-dbserver can be explicitly specified. It is dynamic
# unless explicitly specified.

# phpmyadmin_port: "8036"
# phpmyadmin_https_port: "8037"
# The PHPMyAdmin ports can be changed from the default 8036 and 8037

# mailhog_port: "8025"
# mailhog_https_port: "8026"
# The MailHog ports can be changed from the default 8025 and 8026

# webimage_extra_packages: [php7.4-tidy, php-bcmath]
# Extra Debian packages that are needed in the webimage can be added here

# dbimage_extra_packages: [telnet,netcat]
# Extra Debian packages that are needed in the dbimage can be added here

# use_dns_when_possible: true
# If the host has internet access and the domain configured can
# successfully be looked up, DNS will be used for hostname resolution
# instead of editing /etc/hosts
# Defaults to true

# project_tld: ddev.site
# The top-level domain used for project URLs
# The default "ddev.site" allows DNS lookup via a wildcard
# If you prefer you can change this to "ddev.local" to preserve
# pre-v1.9 behavior.

# ngrok_args: --subdomain mysite --auth username:pass
# Provide extra flags to the "ngrok http" command, see
# https://ngrok.com/docs#http or run "ngrok http -h"

# disable_settings_management: false
# If true, ddev will not create CMS-specific settings files like
# Drupal's settings.php/settings.ddev.php or TYPO3's AdditionalConfiguration.php
# In this case the user must provide all such settings.

# You can inject environment variables into the web container with:
# web_environment:
# - SOMEENV=somevalue
# - SOMEOTHERENV=someothervalue

# no_project_mount: false
# (Experimental) If true, ddev will not mount the project into the web container;
# the user is responsible for mounting it manually or via a script.
# This is to enable experimentation with alternate file mounting strategies.
# For advanced users only!

# Many ddev commands can be extended to run tasks before or after the
# ddev command is executed, for example "post-start", "post-import-db",
# "pre-composer", "post-composer"
# See https://ddev.readthedocs.io/en/stable/users/extending-commands/ for more
# information on the commands that can be extended and the tasks you can define
# for them. Example:
#hooks:
# post-import-db:
#   - exec: drush cr
#   - exec: drush updb
```

**Comment by rfay** [comment]
Hi @deviantintegral - Thanks for taking this for a spin!

Each case that I've seen like this has involved a different mutagen version (or instance) running on the computer. 

Could you please `ddev poweroff` and  `killall mutagen`  and give it another try? 

In the alpha2 release, coming shortly, no MUTAGEN_DATA_DIRECTORY is explicitly specified, so that may make it more compatible with running instances of mutagen.

**Comment by deviantintegral** [comment]
I don't have any other mutagen instances running, but I do have mutagen itself installed via homebrew. According to activity monitor `/Users/andrew/.ddev/.mutagen/bin/mutagen` is being used, so at least we know it's not a path issue.

I'm not going to be able to dig further for another week or so, but I'll take a look at alpha2 when it's out.

**Comment by rfay** [comment]
Alpha2 is out, thanks. And try `killall mutagen` (macOS or Linux) to see what might happen. Thanks for taking the time with this!

**Comment by rfay** [comment]
I'm going to say this is fixed in [v1.18.0-alpha3](https://github.com/drud/ddev/releases/tag/v1.18.0-alpha3) but I sure hope you'll give it another shot! Closing this for now, but happy to continue the conversation here or elsewhere, and happy to reopen this if needed.
